lisa/tests/scheduler/eas_behaviour.py

```diff
@@ -201,15 +223,15 @@ def _from_target(cls, target: Target, *, res_dir: ArtifactPath = None, collector
         :meth:`lisa.tests.base.RTATestBundle.get_rtapp_profile`
         """
         plat_info = target.plat_info
-        rtapp_profile = cls.get_rtapp_profile(plat_info)
+        rtapp_profile = cls.get_rtapp_profile(plat_info, big_task_duty_cycle=big_task_duty_cycle)
```

Comment:
kwargs can be factorized to prevent them from getting out of sync:
```
profile_kwargs = dict(big_task_duty_cycle=big_task_duty_cycle)

...

cls.get_rtapp_profile(plat_info, **profile_kwargs)

...

cls(res_dir, plat_info, rtapp_profile_kwargs=profile_kwargs)
```